force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -121,7 +145,8 @@ public with sharing class VariablesDataTypesOperators {
 	 * @return The complete sentence as a single String.
 	 */
 	public static String createSentence(String noun, String verb, String endingPunctuation) {
-		return null; // Replace null with the variable you used to store the result
+        String sentance;
+        return sentance = 'The '+ noun + ' is ' + verb + endingPunctuation; 
```

Comment:
For formatting you often want to set the variables and then return it without operation acting on it.
This works but it's for cleaner formatting.
```suggestion
        String sentence;
        sentance = 'The '+ noun + ' is ' + verb + endingPunctuation; 
        return sentence;
```